### ISSUE TITLE
Errors while running evaluation

### ISSUE THREAD

**Issue by ma-siddiqui** [leebebeto/Inter-Prototype#4]
Hi,

I am facing below errors while running the pre-trained models.

**Exception has occurred: RuntimeError (note: full exception trace is shown but execution is paused at: _run_module_as_main)
mat1 dim 1 must match mat2 dim 0**

  File "/home/ajmal/anaconda3/envs/leebebeto_face_dev/lib/python3.7/site-packages/torch/nn/functional.py", line 1753, in linear
    return torch._C._nn.linear(input, weight, bias)
  File "/home/ajmal/anaconda3/envs/leebebeto_face_dev/lib/python3.7/site-packages/torch/nn/modules/linear.py", line 94, in forward
    return F.linear(input, self.weight, self.bias)
  File "/home/ajmal/anaconda3/envs/leebebeto_face_dev/lib/python3.7/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/ajmal/anaconda3/envs/leebebeto_face_dev/lib/python3.7/site-packages/torch/nn/modules/container.py", line 119, in forward
    input = module(input)
  File "/home/ajmal/anaconda3/envs/leebebeto_face_dev/lib/python3.7/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)

Thanks,

**Comment by ma-siddiqui** [comment]
sorted out.
